### PR TITLE
Fix composer install

### PR DESCRIPTION
**Pull request by Maks3w**
Tuning composer.json allow install out of the box

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -42,6 +42,8 @@
             "src/"
         ]
     },
+    "minimum-stability": "dev",
```
Comment (GrahamCampbell) [inline composer.json]: why is this needed?

Comment (Maks3w) [inline composer.json]: Allow me rewrite your question

Why both settings are needed?

It download the sources when is not possible resolve an stable version (when circular dependencies occur)

Comment (GrahamCampbell) [inline composer.json]: no - the question was why isn't stable good enough?

Comment (Maks3w) [inline composer.json]: Again. Both settings works in tandem.

https://travis-ci.org/sebastianbergmann/php-code-coverage/jobs/87362951#L102-L149

Comment (GrahamCampbell) [inline composer.json]: massive :-1: for this change

Comment (GrahamCampbell) [inline composer.json]: your link there answers my question with, no, it's not needed, and shouldn't be added for no reason

**Comment by sebastianbergmann** [comment]
Makes no sense to me. And it works for me without this change.
